homeassistant/components/obihai/sensor.py

```diff
@@ -32,7 +34,7 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
You are now doing sync I/O in async context, this needs to be handled.
Have a look here <https://developers.home-assistant.io/docs/asyncio_working_with_async#calling-sync-functions-from-async>